### PR TITLE
docs: clarify the JDK update steps on MacOS

### PR DESCRIPTION
**Pull request by petetnt**
This PR adds ELI5 style installation steps for installing the `Azul Zulu` JDK on MacOS, as glancing through it was not entirely obvious that you actually need to navigate to the folder and install the JDK manually, at least for me.

### CODE REVIEW

**Comment by petetnt** [comment]
At least for me, `brew install --cask zulu@17` only added the `.pkg` in the said folder and I had to go and install it manually.

**Review by cortinico — APPROVED** [review]

File: docs/_getting-started-macos-android.md [hunk 1]
```diff
@@ -24,11 +24,22 @@ We recommend installing the OpenJDK distribution called Azul **Zulu** using [Hom
...
 brew info --cask zulu@17
+
+# ==> zulu@17: <version number>
+# https://www.azul.com/downloads/
+# Installed
+# /opt/homebrew/Caskroom/zulu@17/<version number> (185.8MB) (note that the path is /usr/local/Caskroom on non-Apple Silicon Macs)
+# Installed using the formulae.brew.sh API on 2024-06-06 at 10:00:00
+
+# Navigate to the folder
+finder /opt/homebrew/Caskroom/zulu@17/<version number> # or /usr/local/Caskroom/zulu@17/<version number>
```
Comment (fxlui) [inline docs/_getting-started-macos-android.md:37]: `finder` did not work for me but `open` did
Screenshot 2024-10-27 at 2 36 18 PM
